index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">

```

Comment:
use spaces instead of tabs.

Reply:
https://stackoverflow.blog/2017/06/15/developers-use-spaces-make-money-use-tabs/

Reply:
spaces  ===  tabs +2,4 year experience, thx, such a very interesting article and amazing data